services/connectivity/src/persistence/connectivity.js

```diff
@@ -20,16 +20,13 @@ module.exports = (
    * @returns {Object[]} Values of area distribution for each category of protected area
    * connectivity
    */
-  findCurrentPAConnectivity: async (areaType, areaId, year = 2020) => {
-    try {
-      return connectivity.query()
-        .where({ geofence_type: areaType, geofence_id: areaId, prot_year: year })
-        .select('protconn', 'protunconn', 'unprotected', 'area_ha');
-    } catch (e) {
+  findCurrentPAConnectivity: async (areaType, areaId, year = 2020) => connectivity.query()
```

Comment:
Put the function body in another line, see these [examples](https://github.com/airbnb/javascript#arrows--one-arg-parens) (the good sections)

Reply:
Apply this for all functions

Reply:
Also you should delete the `async`

Reply:
see `findTotalNumberOfSpecies` in `richness_nos.js` as reference